src/transformers/training_args.py

```diff
@@ -1393,6 +1397,10 @@ def __post_init__(self):
         if self.disable_tqdm is None:
             self.disable_tqdm = logger.getEffectiveLevel() > logging.WARN
 
+        # Deal with alias
+        if self.eval_strategy is not None:
+            self.evaluation_strategy = self.eval_strategy
```

Comment:
We need to deal with both `evaluation_strategy` and `eval_strategy` being set (even if very unlikely)